packages/website-eslint/src/linter/CompilerHost.js

```diff
@@ -0,0 +1,48 @@
+import { getDefaultLibFileName } from 'typescript';
```

Comment:
question: could we leverage [`@typescript/vfs`](https://www.npmjs.com/package/@typescript/vfs) for some of this infra?

Reply:
we could, but i will prefer to do this after initial code is merged, 

generally I'd like to remove ts compilation from this code, and reuse AST from sandbox, but that's going to require more changes

Reply:
as for now we are actually using `@typescript/vfs` but not in so obvious way

1. we are compiling lib with typescript as external library
2. we are using typescript as window.ts
3. windows.ts is provided by sandbox

Reply:
Sure thing! Happy to leave this as a follow up.